packages/explorer/src/bin/explorer.ts

```diff
@@ -97,6 +101,30 @@ async function startExplorer() {
   }
 }
 
+async function startStoreIndexer() {
+  if (chainId !== anvil.id) {
+    console.log("Skipping SQLite indexer for non-anvil chain ID", chainId);
+    return;
+  }
+
+  if (existsSync(indexerDatabasePath)) {
+    execSync(`shx rm -rf ${indexerDatabasePath}`);
```

Comment:
since we're in node, can we use `fs.rm(indexerDatabasePath, {recursive:true, force:true})`

Reply:
also been trying to use fs/promises where I can rather than regular fs because it lets you parallelize when you need it

Reply:
Good point, will keep that in mind. In this case I'm not sure if async version makes sense since we'd have to wait for the database file to be deleted anyway, or am I missing something? So changed to `fs.rmSync`.

Reply:
you can still `await`, I just find it useful to have the promise variants handy when you do want to parallelize certain operations

Reply:
interesting, didn't think about that but I see your point, updated